features/step_definitions/service_steps.rb

```diff
@@ -2,6 +2,14 @@
   provider.services.create! :name => name, :mandatory_app_key => false
 end
 
+Given /^a backend api "([^"]*)")$/ do |name|
+  @provider.backend_apis.create!(:name => name, :private_endpoint => 'https://foo')
```

Comment:
just 💅 

but I prefer the `name: name, private_endpoint: 'https://foo'` syntax.. wdyt?

Reply:
my bad, sorry, because and copied&pasted from another one :blush: